lib/twilio/device.ts

```diff
@@ -1717,6 +1733,71 @@ namespace Device {
      */
     edge?: string[] | string;
 
+    /**
```

Comment:
I manually generated and confirmed the autogenerated API docs for this option look good. Perhaps too verbose, wdyt @charliesantos?